style.css

```diff
@@ -373,13 +378,114 @@ html {
   scroll-behavior: smooth;
 }
 
+.flex-reverse {
+  flex-direction: row-reverse;
+}
+
 @media screen and (min-width: 768px) {
   #skills {
     width: 60%;
     padding: 142px 142px 0 0;
   }
 
-  .reverse > .card-body {
+  .modal {
+    height: 100vh;
+    position: fixed;
+    right: 0;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    background-color: rgb(193, 199, 208, 0.7);
+    padding: 10px 142px 0 142px;
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
- [ ] **[Optional] Good job so far 👏 👏** however, It would be better to align the pop-up window in the center of the page for the best user experience 🙏 

_You may watch this [video](https://www.youtube.com/watch?v=mVYgtzDLZfY) on 4 easy ways to put a div in the center 😎😎😎_

Reply:
Fixed.